content/platforms.md

```diff
@@ -110,12 +110,12 @@ versions for Chef Infra Client:
 </tr>
 <tr class="even">
 <td>SUSE Enterprise Linux Server</td>
-<td><code>x86_64</code>, <code>s390x</code> (12.x only)</td>
```

Comment:
We should probably have an Architecture EOL table at the bottom of this page. It would also include when we dropped PowerPC support for Chef Infra Server.

Reply:
That seems doing, but outside the scope of this. We'd need to go back and figure out when we dropped the various archs

Reply:
Can be separate, sure, but we're removing s390x here with no mention of where it went.

Reply:
Made an issue for it: #2455 2455